### PR TITLE
fix [SPARK-740] block manage UI throws exception when enabling Spark Streaming

### PR DESCRIPTION
**Pull request by jerryshao**
Because of Spark Streaming's input block naming format (input-xx-xx) is different from Spark (rdd_xx_xx), so block manager UI will throw exception when group this block using block name, detailed thing is illustrated in [[SPARK-740]](https://spark-project.atlassian.net/browse/SPARK-740).

Currently I add a key filter to filter out all the Spark Streaming's input block RDD, so the UI will only display processed RDD. Also I sort RDDInfo by RDD's id, so it is clear to see in the UI.

### CODE REVIEW

File: core/src/main/scala/spark/storage/StorageUtils.scala [hunk 1]
```diff
@@ -22,35 +22,41 @@ case class StorageStatus(blockManagerId: BlockManagerId, maxMem: Long,
...
     sc: SparkContext) : Array[RDDInfo] = {
 
     // Group by rddId, ignore the partition name
-    val groupedRddBlocks = infos.groupBy { case(k, v) =>
-      k.substring(0,k.lastIndexOf('_'))
-    }.mapValues(_.values.toArray)
+    val groupedRddBlocks = infos.filterKeys(_.startsWith("rdd_")).
+      groupBy { case(k, v) =>
+        k.substring(0,k.lastIndexOf('_'))
+      }.mapValues(_.values.toArray)
```
Comment (mateiz) [inline core/src/main/scala/spark/storage/StorageUtils.scala]: To match the style of the rest of the codebase, format this like this:

```
val groupedRddBlocks = infos.filterKeys(_.startsWith("rdd_")).groupBy {
  ...
}.mapValues(...)
```

**Comment by mateiz** [comment]
Looks great, thanks! I just made one small comment on formatting.

**Comment by tdas** [comment]
I have taken a look as well. Looks good.

**Comment by jerryshao** [comment]
I've changed the code formatting, please review, Thanks.

**Comment by mateiz** [comment]
Great, thanks.

**Pull request merged**